BlueDBInMemory/src/main/java/io/bluedb/memory/BlueDbInMemory.java

```diff
@@ -1,21 +1,29 @@
 package io.bluedb.memory;
 
 import java.io.Serializable;
+import java.util.HashMap;
+import java.util.Map;
 import io.bluedb.api.BlueDb;
 import io.bluedb.api.exceptions.BlueDbException;
 import io.bluedb.api.BlueCollection;
 
 public class BlueDbInMemory implements BlueDb {
 
+	@SuppressWarnings("rawtypes")
+	Map<Class, BlueCollection> collections = new HashMap<>();
```

Comment:
You could declare the key type to be Class<?> instead of ignoring the warning. Or you could do Class<? extends Serializable>. Doesn't really matter much.